doc/orchestrator.rst

```diff
@@ -1,207 +1,725 @@
+    exe_ex = sys.executable
+    # Initialize a logger object
+    logger = get_logger("Example Experiment Log")
+    # Initialize the Experiment
+    exp = Experiment("getting-started", launcher="auto")
+
+Colocated Model Initialization
+------------------------------
+In the next stage of the experiment, we
+create and launch a colocated ``Model`` that
+runs the application script with a ``Orchestrator``
+on the same compute node.
 
-.. code-block:: bash
+Step 1: Configure
+'''''''''''''''''
+In this example experiment, the ``Model`` application is a Python script as defined in section:
+:ref:`Application Script<colocated_orch_app_script>`. Before initializing the ``Model`` object, we must use
+``Experiment.create_run_settings()`` to create a ``RunSettings`` object that defines how to execute
+the ``Model``. To launch the Python script in this example workflow, we specify the path to the application
+file `application_script.py` as the `exe_args` parameter and the executable `exe_ex` (the Python
+executable on this system) as `exe` parameter. The ``Experiment.create_run_settings()`` function
+will return a ``RunSettings`` object that can then be used to initialize the ``Model`` object.
 
-  # build KeyDB
-  # see https://github.com/EQ-Alpha/KeyDB
+.. note::
+  Change the `exe_args` argument to the path of the application script
+  on your file system to run the example.
 
-  # get KeyDB configuration file
-  wget https://github.com/CrayLabs/SmartSim/blob/d3d252b611c9ce9d9429ba6eeb71c15471a78f08/smartsim/_core/config/keydb.conf
+Use the ``RunSettings`` helper functions to
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the ``Model`` to run on a single compute node.
 
-  export REDIS_PATH=/path/to/keydb-server
-  export REDIS_CONF=/path/to/keydb.conf
+.. code-block:: python
 
-  # run smartsim workload
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/path/to/clustered_model.py")
+    # Configure RunSettings object
+    model_settings.set_nodes(1)
 
+Step 2: Initialize
+''''''''''''''''''
+Next, create a ``Model`` instance using the ``Experiment.create_model()`` factory method.
+Pass the ``model_settings`` object as an argument to the method and
+assign the returned ``Model`` instance to the variable `model`:
+
+.. code-block:: python
+
+    # Initialize a SmartSim Model
+    model = exp.create_model("colo_model", model_settings)
+
+Step 3: Colocate
+''''''''''''''''
+To colocate the `model`, use the ``Model.colocate_db_uds()`` function.
+This function will colocate an ``Orchestrator`` instance with this ``Model`` over
+a Unix domain socket connection.
+
+.. code-block:: python
+
+    # Colocate the Model
+    model.colocate_db_uds()
+
+Step 4: Generate Files
+''''''''''''''''''''''
+Next, generate the ``Experiment`` entity output files by passing the ``Model`` instance to
```

Comment:
Next, generate the ``Experiment`` entity output files by passing the ``Model`` instance to

->

Next, generate the ``Experiment`` entity directories by passing the ``Model`` instance to